### ISSUE TITLE
the loss stopped dropping at around 2.5, can your give me some suggestion

### ISSUE THREAD

**Issue by xunkaixin** [tensorflow/models#1825]
System information

OS Platform and Distribution: Linux Ubuntu 16.04
TensorFlow installed from: source
TensorFlow version: 1.1.0
Bazel version: compiling from source, using bazel release 0.4.5
Describe the problem

on my own dataset  the loss stopped dropping at around 2.5

Source code / logs
 the learning rate is 0.004 and change it to 0.0004， almost no difference

INFO:tensorflow:global step 14356: loss = 3.0286 (0.69 sec/step)
INFO:tensorflow:global step 14357: loss = 2.3087 (0.70 sec/step)
INFO:tensorflow:global step 14358: loss = 2.9221 (0.71 sec/step)
INFO:tensorflow:global step 14359: loss = 2.4507 (0.70 sec/step)
INFO:tensorflow:global step 14360: loss = 2.6003 (0.71 sec/step)
INFO:tensorflow:global step 14361: loss = 2.6226 (0.68 sec/step)
INFO:tensorflow:global step 14362: loss = 2.3599 (0.72 sec/step)
INFO:tensorflow:global step 14363: loss = 2.4288 (0.72 sec/step)
INFO:tensorflow:global step 14364: loss = 2.5999 (0.74 sec/step)
INFO:tensorflow:global step 14365: loss = 2.5063 (0.71 sec/step)
INFO:tensorflow:global step 14366: loss = 2.7320 (0.68 sec/step)
INFO:tensorflow:global step 14367: loss = 2.8757 (0.66 sec/step)
INFO:tensorflow:global step 14368: loss = 2.5452 (0.74 sec/step)
INFO:tensorflow:global step 14369: loss = 2.1440 (0.69 sec/step)
INFO:tensorflow:global step 14370: loss = 2.9030 (0.71 sec/step)
INFO:tensorflow:global step 14371: loss = 2.5339 (0.70 sec/step)
INFO:tensorflow:global step 14372: loss = 2.2061 (0.72 sec/step)
INFO:tensorflow:global step 14373: loss = 2.2826 (0.70 sec/step)
INFO:tensorflow:global step 14374: loss = 3.3629 (0.69 sec/step)
INFO:tensorflow:global step 14375: loss = 2.7311 (0.69 sec/step)
INFO:tensorflow:global step 14376: loss = 2.5831 (0.70 sec/step)
INFO:tensorflow:global step 14377: loss = 2.8963 (0.72 sec/step)
INFO:tensorflow:global step 14378: loss = 3.2740 (0.70 sec/step)

**Comment by DanMossa** [comment]
Did you end up figuring out the issue?
